### ISSUE TITLE
All calls to oxauth fails when httpLoggingEnabled is set to true

### ISSUE THREAD

**Issue by syntrydy** [GluuFederation/oxAuth#914]
# Description

When the parameter **httpLoggingEnabled** is set to **true**, all scim call fail.

# Oxauth partial log:

```
2018-10-01 07:26:16,067 INFO  [qtp1744347043-11] [org.xdi.oxauth.auth.Authenticator] (Authenticator.java:217) - Authentication success for Client: '@!191F.F9DE.AF55.F2DC!0001!F8AF.E203!0008!F29D.C732'
2018-10-01 07:26:16,099 ERROR [qtp1744347043-10] [xdi.oxauth.uma.ws.rs.UmaPermissionRegistrationWS] (UmaPermissionRegistrationWS.java:134) - Failed to parse uma permission request
java.io.EOFException: No content to map to Object due to end of input
        at org.codehaus.jackson.map.ObjectMapper._initForReading(ObjectMapper.java:2775) ~[jackson-mapper-asl-1.9.11.jar:1.9.11]
        at org.codehaus.jackson.map.ObjectMapper._readMapAndClose(ObjectMapper.java:2718) ~[jackson-mapper-asl-1.9.11.jar:1.9.11]
        at org.codehaus.jackson.map.ObjectMapper.readValue(ObjectMapper.java:1863) ~[jackson-mapper-asl-1.9.11.jar:1.9.11]
        at org.xdi.oxauth.uma.ws.rs.UmaPermissionRegistrationWS.parseRequest(UmaPermissionRegistrationWS.java:128) [classes/:?]
        at org.xdi.oxauth.uma.ws.rs.UmaPermissionRegistrationWS.registerPermission(UmaPermissionRegistrationWS.java:93) [classes/:?]
        at org.xdi.oxauth.uma.ws.rs.UmaPermissionRegistrationWS$Proxy$_$$_WeldClientProxy.registerPermission(Unknown Source) [classes/:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_162]
```

# Oxtrust partial log
```
- Authorization header not found
2018-10-01 07:26:16,084 INFO  [qtp1744347043-11] [org.gluu.oxtrust.service.uma.ScimUmaProtectionService] (ScimUmaProtectionService.java:118)
 - SCIM is protected by UMA
2018-10-01 07:26:16,104 ERROR [qtp1744347043-11] [org.gluu.oxtrust.service.uma.ScimUmaProtectionService] (ScimUmaProtectionService.java:127)
 - RESTEASY003150: Error status 400 Bad Request returned
org.jboss.resteasy.client.ClientResponseFailure: RESTEASY003150: Error status 400 Bad Request returned
        at org.jboss.resteasy.client.core.BaseClientResponse.createResponseFailure(BaseClientResponse.java:584) ~[resteasy-jaxrs-3.0.24.Fina
l.jar:3.0.24.Final]
        at org.jboss.resteasy.client.core.BaseClientResponse.createResponseFailure(BaseClientResponse.java:575) ~[resteasy-jaxrs-3.0.24.Fina
l.jar:3.0.24.Final]
        at org.jboss.resteasy.client.core.BaseClientResponse.checkFailureStatus(BaseClientResponse.java:569) ~[resteasy-jaxrs-3.0.24.Final.j
ar:3.0.24.Final]
        at org.jboss.resteasy.client.core.extractors.BodyEntityExtractor.extractEntity(BodyEntityExtractor.java:39) ~[resteasy-jaxrs-3.0.24.
Final.jar:3.0.24.Final]

```

**Comment by yuriyz** [comment]
oxTrust
```
2018-10-03 05:12:02,945 INFO  [qtp1744347043-11] [org.gluu.oxtrust.service.uma.ScimUmaProtectionService] (ScimUmaProtectionService.java:107) - ==== SCIM Service call intercepted ====
2018-10-03 05:12:02,945 INFO  [qtp1744347043-11] [org.gluu.oxtrust.service.uma.ScimUmaProtectionService] (ScimUmaProtectionService.java:108) - Authorization header not found
2018-10-03 05:12:04,023 INFO  [qtp1744347043-11] [org.gluu.oxtrust.service.uma.ScimUmaProtectionService] (ScimUmaProtectionService.java:118) - SCIM is protected by UMA
2018-10-03 05:12:04,029 INFO  [qtp1744347043-11] [org.gluu.oxtrust.service.uma.UmaPermissionService] (UmaPermissionService.java:82) - ##### Initializing custom ClientExecutor...
2018-10-03 05:12:04,106 INFO  [qtp1744347043-11] [org.gluu.oxtrust.service.uma.UmaPermissionService] (UmaPermissionService.java:92) - ##### Initializing custom ClientExecutor DONE
2018-10-03 05:12:04,200 ERROR [qtp1744347043-11] [org.gluu.oxtrust.service.uma.ScimUmaProtectionService] (ScimUmaProtectionService.java:127) - RESTEASY003150: Error status 400 Bad Request returned
org.jboss.resteasy.client.ClientResponseFailure: RESTEASY003150: Error status 400 Bad Request returned
	at org.jboss.resteasy.client.core.BaseClientResponse.createResponseFailure(BaseClientResponse.java:584) ~[resteasy-jaxrs-3.0.24.Final.jar:3.0.24.Final]
	at org.jboss.resteasy.client.core.BaseClientResponse.createResponseFailure(BaseClientResponse.java:575) ~[resteasy-jaxrs-3.0.24.Final.jar:3.0.24.Final]
	at org.jboss.resteasy.client.core.BaseClientResponse.checkFailureStatus(BaseClientResponse.java:569) ~[resteasy-jaxrs-3.0.24.Final.jar:3.0.24.Final]
	at org.jboss.resteasy.client.core.extractors.BodyEntityExtractor.extractEntity(BodyEntityExtractor.java:39) ~[resteasy-jaxrs-3.0.24.Final.jar:3.0.24.Final]
	at org.jboss.resteasy.client.core.ClientInvoker.invoke(ClientInvoker.java:128) ~[resteasy-jaxrs-3.0.24.Final.jar:3.0.24.Final]
	at org.jboss.resteasy.client.core.ClientProxy.invoke(ClientProxy.java:89) ~[resteasy-jaxrs-3.0.24.Final.jar:3.0.24.Final]
	at com.sun.proxy.$Proxy173.registerPermission(Unknown Source) ~[?:?]
	at org.gluu.oxtrust.service.uma.UmaPermissionService.registerResourcePermission(UmaPermissionService.java:173) ~[classes/:?]
	at org.gluu.oxtrust.service.uma.UmaPermissionService.prepareRegisterPermissionsResponse(UmaPermissionService.java:184) ~[classes/:?]
	at org.gluu.oxtrust.service.uma.UmaPermissionService.validateRptToken(UmaPermissionService.java:135) ~[classes/:?]
	at org.gluu.oxtrust.service.uma.UmaPermissionService$Proxy$_$$_WeldClientProxy.validateRptToken(Unknown Source) ~[classes/:?]
	at org.gluu.oxtrust.service.uma.BaseUmaProtectionService.processUmaAuthorization(BaseUmaProtectionService.java:151) ~[classes/:?]
	at org.gluu.oxtrust.service.uma.ScimUmaProtectionService.processAuthorization(ScimUmaProtectionService.java:119) [classes/:?]
	at org.gluu.oxtrust.service.uma.ScimUmaProtectionService$Proxy$_$$_WeldClientProxy.processAuthorization(Unknown Source) [classes/:?]
	at org.gluu.oxtrust.service.filter.AuthorizationProcessingFilter.filter(AuthorizationProcessingFilter.java:80) [classes/:?]
	at org.gluu.oxtrust.service.filter.AuthorizationProcessingFilter$Proxy$_$$_WeldClientProxy.filter(Unknown Source) [classes/:?]
	at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTarget(ResourceMethodInvoker.java:263) [resteasy-jaxrs-3.0.24.Final.jar:3.0.24.Final]
	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:249) [resteasy-jaxrs-3.0.24.Final.jar:3.0.24.Final]
	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:236) [resteasy-jaxrs-3.0.24.Final.jar:3.0.24.Final]
	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:406) [resteasy-jaxrs-3.0.24.Final.jar:3.0.24.Final]
	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:213) [resteasy-jaxrs-3.0.24.Final.jar:3.0.24.Final]
	at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:228) [resteasy-jaxrs-3.0.24.Final.jar:3.0.24.Final]
	at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:56) [resteasy-jaxrs-3.0.24.Final.jar:3.0.24.Final]
	at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:51) [resteasy-jaxrs-3.0.24.Final.jar:3.0.24.Final]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790) [servlet-api-3.1.jar:3.1.0]
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:864) [jetty-servlet-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1655) [jetty-servlet-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.websocket.server.WebSocketUpgradeFilter.doFilter(WebSocketUpgradeFilter.java:215) [websocket-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1642) [jetty-servlet-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.ocpsoft.rewrite.servlet.RewriteFilter.doFilter(RewriteFilter.java:226) [rewrite-servlet-3.4.2.Final.jar:3.4.2.Final]
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1642) [jetty-servlet-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:533) [jetty-servlet-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:146) [jetty-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:548) [jetty-security-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132) [jetty-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257) [jetty-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1595) [jetty-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:255) [jetty-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1253) [jetty-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:203) [jetty-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:473) [jetty-servlet-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1564) [jetty-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:201) [jetty-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1155) [jetty-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144) [jetty-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:219) [jetty-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:126) [jetty-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132) [jetty-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.server.Server.handle(Server.java:531) [jetty-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:352) [jetty-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:260) [jetty-server-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:281) [jetty-io-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:102) [jetty-io-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118) [jetty-io-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:754) [jetty-util-9.4.9.v20180320.jar:9.4.9.v20180320]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:672) [jetty-util-9.4.9.v20180320.jar:9.4.9.v20180320]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_162]

```

**Comment by yuriyz** [comment]
fixed in 3.1.5 and merged into master.
